### DISCUSSION TITLE
如果內容需要刪修，請在此串回報

### DISCUSSION THREAD

**Discussion by jim60105** [sound-buttons/sound-buttons_configs#3] [General]
## 如果需要刪修，或對內容、分類有建議，歡迎給我留言！

**Comment by leadra** [comment]
涼風しとら
糟糕...傳錯音檔了
需要刪除>>你的豆花30塊
今天怎麼上傳youtu或是用套剪輯，一直失敗上傳錯誤...
所以放音檔，好像音檔就不能附上來源了耶

**Reply by leadra** [reply]
00:27:21 你們這些~處生00:27:43
https://youtu.be/J6dF-YTwqWs?t=1641
目前用網址都上傳錯誤失敗~(O:O)

**Reply by leadra** [reply]
你的豆花30塊
有重新傳新的，舊的需要刪除

有投稿一些，審核再麻煩你了

後面我可以在json新增YT來源嗎?
沒法放線上剪輯，全部都是音檔

**Reply by leadra** [reply]
![image](https://user-images.githubusercontent.com/67009996/210571280-a7d50b45-f37e-4844-9b58-cb5680f93eca.png)
上傳時，沒選到炒飯區，不好意思，需要移動

**Reply by jim60105** [reply]
修正了，請再確認
如果需要大量修正可以發PR給我
要稍微注意一下的是，這個repo裡的檔案是正式那邊的版本，如果跟待審核的版本落差太大我會不好合併
建議不要兩邊一起改

**Reply by leadra** [reply]
好的，主要是想增加音檔的來源
"videoId": "",
感謝更新

**Comment by nekoneko2023** [comment]
sitora肚子餓了 
最下面的那個傳錯了~請幫忙刪除 感謝

**Reply by jim60105** [reply]
已處理

**Comment by leadra** [comment]
https://button.sound-buttons.maki0419.com/shitora/醒醒好嗎
不好意思，聲音剪完有短少，要重新，想麻煩幫我刪除

**Reply by jim60105** [reply]
已刪除

**Comment by zeldaluna** [comment]
不好意思 東域艾瑞兒的"我又卡住XK7" 打太快按到Enter就送出去了 還麻煩協助刪除 感謝🥺

**Reply by jim60105** [reply]
OK

**Comment by leadra** [comment]
https://sound-buttons.click/shitora/b369c827-6399-4481-ba9e-a76ae3a84aa0
![image](https://github.com/sound-buttons/sound-buttons_configs/assets/67009996/bb81fb45-a774-49c2-b27d-b3b41388e865)
不好意思，多傳錯一個空白的按鈕了

**Reply by jim60105** [reply]
OK

**Comment by syachiku0220** [comment]
<img width="352" height="211" alt="gp" src="https://github.com/user-attachments/assets/2522093f-fc9b-4647-8bb8-b7b760a0bb89" />
不好意思之前分組名字没改，沿用了V名字
想更改成按配信日分組，分組 花園さやか->12月9日，麻煩大大

**Reply by jim60105** [reply]
改好了

**Reply by syachiku0220** [reply]
謝謝
